draft-ietf-rtcweb-jsep.xml

```diff
@@ -3468,7 +3468,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
                         <t>If present, a single "a=rtcp" attribute MUST
                         be parsed as specified in <xref target="RFC3605"
-                        />, Section 2.1, but its value is ignored.</t>
+                        />, Section 2.1, but its value is ignored as it is
+                        overridden by the RTCP information carried in a=candidate lines.</t>
```

Comment:
I would say instead "is ignored as the a= candidate lines contain the RTCP information".